app/views/projects/show.html.erb

```diff
@@ -0,0 +1,4 @@
+<h1><%= @project.name %><h1>
+<p><%= @project.material %></p>
+<p><%= @challenge.theme %></p>
+<p>Number of Contestants: <%=@project.contestants.count%></p>
```

Comment:
This would be considered an MVC infraction, since `.count` alters the constructed SQL that AR sends to the database (use `.to_sql` in a pry session to see what I mean). Also, `.count` is a calculation, and since we want to be sure we display only _tested_ calculations to our end users, we should put this in a model method and write a model test for it as well.